cachito/web/api_v1.py

```diff
@@ -472,14 +472,17 @@ def patch_request(request_id):
 
     db.session.commit()
 
-    bundle_dir = RequestBundleDir(request.id, root=flask.current_app.config["CACHITO_BUNDLES_DIR"])
+    bundle_dir: RequestBundleDir = RequestBundleDir(
```

Comment:
Do we need the type annotation for RequestBundleDir? I'd say the `RequestBundleDir()` call makes the type pretty clear :-)

Reply:
Oh, this type annotation is optional. IIRC, it just makes my editor happy to give auto-complete candidates.

Reply:
I see, we can keep it then